src/common.jl

```diff
@@ -24,13 +25,19 @@ TaylorMethod() = error("Maximum order must be specified for the Taylor method")
 export TaylorMethod
 
 function DiffEqBase.solve(
-    prob::DiffEqBase.AbstractODEProblem{uType,tType,isinplace},
+    prob::DiffEqBase.AbstractODEProblem{uType,tupType,isinplace},
     alg::AlgType,
     timeseries=[],ts=[],ks=[];
-    verbose=true, abstol = 1e-6, save_start = true,
+    verbose=true,
+    saveat=eltype(tupType)[],
+    abstol = 1e-6,
+    save_start =  isempty(saveat) || saveat isa Number || prob.tspan[1] in saveat,
+    save_end   =  isempty(saveat) || saveat isa Number || prob.tspan[2] in saveat,
     timeseries_errors=true, maxiters = 1_000_000,
     callback=nothing, kwargs...) where
-        {uType, tType, isinplace, AlgType <: TaylorAlgorithm}
+        {uType, tupType, isinplace, AlgType <: TaylorAlgorithm}
+
```

Comment:
Updated the constructed to mimic that used by OrdinaryDiffEq

Reply:
OK